tests/pipelines/eks/awscli-cl2-load-with-addons.yaml

```diff
@@ -15,6 +15,9 @@ spec:
   - name: slack-hook
   - name: slack-message
   - name: amp-workspace-id
+  - name: kubernetes-version
+    default: "1.23"
```

Comment:
I thought we have this default at task level IIRC unless it was removed recently ?

Reply:
yeah its there.

Reply:
Then can we remove it from here ? We don't need this redundancy.

Reply:
removed